dali/operators/reader/nemo_asr_reader_op.cc

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/reader/nemo_asr_reader_op.h"
+
+namespace dali {
+
+DALI_REGISTER_OPERATOR(NemoAsrReader, NemoAsrReader, CPU);
+
+DALI_SCHEMA(NemoAsrReader)
+  .NumInput(0)
+  .NumOutput(1)
+  .DocStr(R"code(Read automatic speech recognition (ASR) data (audio, text) from a 
+NVIDIA NeMo compatible manifest.
+
+Example manifest file::
+
+    {"audio_filepath": "path/to/audio1.wav", "duration": 3.45, "text": "this is a nemo tutorial"}
+    {"audio_filepath": "path/to/audio1.wav", "offset": 3.45, "duration": 1.45, "text": "same audio file but using offset"}
+    {"audio_filepath": "path/to/audio2.wav", "duration": 3.45, "text": "third transcript in this example"}
+
+This reader produces between 1 and 3 outputs:
+
+- Decoded audio data: float, shape=``(audio_length,)``
+
```

Comment:
I don't think you need empty line between bullet points.

Reply:
Will fix